### PR TITLE
New Presence • Repeat

### PR DESCRIPTION
**Pull request by callumok2004**
t1i70 1

### CODE REVIEW

File: websites/R/Repeat/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "service": "Repeat",
+  "description": {
+    "en": "Repeating your favourite tunes daily."
+  },
+  "url": [
+    "www.repeatradio.net",
+    "repeatradio.net",
+    "repeat.pw"
+  ],
+  "version": "2.5.3",
```
Comment (Bas950) [inline websites/R/Repeat/dist/metadata.json:15]: version 1.0.0

File: websites/R/Repeat/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "logo": "https://repeatradio.net/staff/avatars/default.png",
+  "thumbnail": "https://repeatradio.net/staff/covers/presets/1.png",
```
Comment (Bas950) [inline websites/R/Repeat/dist/metadata.json:17]: upload to imgur

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/R/Repeat/dist/metadata.json (hunk 2 above)
Comment (callumok2004) [reply websites/R/Repeat/dist/metadata.json:17]: self hosted not needed

Comment (doomlerd) [reply websites/R/Repeat/dist/metadata.json:17]: It is still not allowed, please upload to imgur.

Comment (callumok2004) [reply websites/R/Repeat/dist/metadata.json:17]: "or on a CDN that gives you permissions to change image files" this is our cdn, and we wont be changing the image file ever

File: websites/R/Repeat/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "en": "Repeating your favourite tunes daily."
+  },
+  "url": [
+    "www.repeatradio.net",
+    "repeatradio.net",
+    "repeat.pw"
+  ],
+  "version": "1.0.0",
+  "logo": "https://imgur.com/Mh7vi7e",
+  "thumbnail": "https://imgur.com/7RjRna8",
```
Comment (Bas950) [inline websites/R/Repeat/dist/metadata.json:17]: use the i.imgur links

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- websites/R/Repeat/presence.ts  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113205&bid=20007464)

**Review by doomlerd — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**